src/Pos/Block/Logic/Event.hs

```diff
@@ -0,0 +1,69 @@
+module Pos.Block.Logic.Event
```

Comment:
I don't like that this module is here. It's not used and probably will never be used by the node itself (because it's generator-specific). We usually separate code which is used by the node and which is auxiliary.